pushtx/src/p2p/client.rs

```diff
@@ -48,21 +48,21 @@ pub fn client(socks_proxy: Option<SocketAddr>, network: crate::Network) -> Clien
         network: network.into(),
         join_handle,
         our_version: VersionMessage {
-            version: 70015,
+            version: 70016,
             services: bitcoin::p2p::ServiceFlags::NONE,
-            timestamp: crate::posix_time() as i64,
+            timestamp: 0,
             receiver: bitcoin::p2p::Address {
-                services: bitcoin::p2p::ServiceFlags::default(),
+                services: bitcoin::p2p::ServiceFlags::NONE,
                 address: [0; 8],
```

Comment:
What is this `[0; 8]`? Does it create 8 zero bytes? If yes, then that should be 16 instead of 8 bytes.
https://developer.bitcoin.org/reference/p2p_networking.html#version

Reply:
The address struct looks like this:

```rust
/// A message which can be sent on the Bitcoin network
#[derive(Clone, PartialEq, Eq, Hash)]
pub struct Address {
    /// Services provided by the peer whose address this is
    pub services: ServiceFlags,
    /// Network byte-order ipv6 address, or ipv4-mapped ipv6 address
    pub address: [u16; 8],
    /// Network port
    pub port: u16,
}
```

It is indeed 16 bytes, but for some reason rust-bitcoin defines that as 8 shorts. So the end result is the same.